packages/valory/skills/abstract_round_abci/handlers.py

```diff
@@ -666,9 +664,6 @@ def _handle_response(
         try:  # validate message contains a valid address
             validator_config = json.loads(message.info)
             self.context.logger.error(validator_config)
-            parse_result = urlparse(validator_config["tendermint_url"])
```

Comment:
Please comment

Reply:
This is a temporary removal, I'll update this with a better logic. Basically this was causing the agent to crash since I was using the ACN node inside the docker compose with the host set to `http://acn:11000` leading to the parse failure

Reply:
Where is this addressed now?